charts/node/values.yaml

```
@@ -332,8 +332,11 @@ node:
       externalTrafficPolicy: Cluster
       type: ClusterIP # or NodePort, LoadBalancer
       annotations: {}
+      rpcPort: 9944
+      # deprecated, use rpcPort
       httpPort: 9933
-      wsPort: 9944
+      # deprecated, use rpcPort
+      wsPort: 9955
```

Can we keep the old value? 
```suggestion
      wsPort: 9944
```
Otherwise it may break someone deployment.

---

This doesn't work as you can't have the same port number twice on a Service object. Either wsPort or rpcPort have to be set to something different. Would yo uprefer if I set wsPort=9944, rpcPort=9955 ?

---

I agree this solution is better and less surprising for migrating from `websocket-rpc=944` to `rpc=9944`. I implemented it in https://github.com/paritytech/helm-charts/pull/241/commits/9e9f48f6a21594a0080161bdbe2202b1047db40d . I hardcode ports for the legacyRpcFlags=true but switch to variabilized ports (wsPort=9955 as default) for legacyRpcFlags=false

---

Thank you, now  we will not break any old deployment if `legacyRpcFlags` is `true`